includes/class-wc-subscriptions-core-plugin.php

```diff
@@ -529,16 +533,9 @@ public function activate_plugin() {
 				update_option( WC_Subscriptions_admin::$option_prefix . '_paypal_debugging_default_set', 'true' );
 			}
 
-			// If this is the first time activating WooCommerce Subscription we want to enable the customer email notifications (default to 3 days before.)
-			if ( '0' === get_option( WC_Subscriptions_Admin::$option_prefix . '_previous_version', '0' ) && false === get_option( WC_Subscriptions_Admin::$option_prefix . WC_Subscriptions_Email_Notifications::$offset_setting_string, false ) ) {
+			// Enable customer notifications by default.
```

Comment:
```suggestion
			// Enable customer notifications by default for new stores.
```
This is only for new stores, right?